src/components/save-your-key-view.tsx

```diff
@@ -68,7 +68,7 @@ export const SaveYourKeyView: React.FC<{
     hideActions={hideActions}
     title={title || 'Save your Secret Key'}
   >
-    <Stack flexGrow={1} spacing="loose">
+    <Stack spacing="loose" flexGrow={1} justifyContent="center" mt="-78px">
```

Comment:
I subtracted out the total header height here so that it is truly centered vertically (as it is currently nested). Not sure if there is currently a better way to keep this in sync across the app rather than hardcoding it?

Reply:
Ok, I'll remove it. Thx.